modules/light-clients/08-wasm/types/export_test.go

```diff
@@ -0,0 +1,60 @@
+package types
+
+import (
+	"github.com/cosmos/cosmos-sdk/codec"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+/*
+	This file is to allow for unexported functions and fields to be accessible to the testing package.
+*/
+
+// MaxWasmSize is the maximum size of a wasm code in bytes.
+const MaxWasmSize = maxWasmSize
```

Comment:
Just a note, this isn't needed. There is an exported function which returns this value - `MaxWasmByteSize()`